### DISCUSSION TITLE
Picarto private stream question

### DISCUSSION THREAD

**Discussion by FallenMans** [streamlink/streamlink#4056] [Q&A] [answered]
Apologies for a silly question like this, but, Is it at all possible to use streamlink for watching Private streams on Picarto if I have the password? I would have thought including the password in the URL like on browsers "picarto.tv/SomeRandomUser/ThisIsThePassword "would work but it still pops up in CMD as a private stream and doesn't load anything. Tried searching around to see if this was answered anywhere but i couldn't come up with anything.

**Comment by mkbloke** [comment, marked as answer]
The only reason private streams are not enabled is because there weren't any known to the plugin author at the time of writing.  It may well be possible to add support for them, but a test case to work with would be necessary.
